agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -98,6 +101,8 @@ func NewCredentialSpecResource(taskARN, region string,
 		ssmClientCreator:        ssmClientCreator,
 		s3ClientCreator:         s3ClientCreator,
 		CredSpecMap:             make(map[string]string),
+		os:                      oswrapper.NewOS(),
```

Comment:
Where are these used?

Reply:
There are used in the handlers within the package.

Reply:
* https://github.com/aws/amazon-ecs-agent/blob/feature/gMSA/agent/taskresource/credentialspec/credentialspec_windows.go#L525
* https://github.com/aws/amazon-ecs-agent/blob/feature/gMSA/agent/taskresource/credentialspec/credentialspec_windows.go#L468